modules/base/tool.go

```diff
@@ -45,8 +45,8 @@ func EncodeSha1(str string) string {
 }
 
 func ShortSha(sha1 string) string {
-	if len(sha1) == 40 {
```

Comment:
We can allow 7-char SHA, does not mean we want to show with 7-char in default, please remove this change.